synapse/models/inet.py

```diff
@@ -741,6 +741,12 @@ def getBaseModels():
                     'doc': 'An IPv4 address block in Classless Inter-Domain Routing (CIDR) notation.',
                     'ex': '1.2.3.0/24'}),
 
+                ('inet:urlredir', {
+                    'subof': 'comp',
+                    'fields': 'src=inet:url,dst=inet:url',
+                    'doc': 'A URL redirect relationship typically via HTTP code 302',
```

Comment:
Should also account for:
- URL shortening / redirection services (bit.ly, goo.gl, twitter / facebook)
- Last-resort model fudging in cases of a web-based exploit chain where part of the chain is missing (e.g., "we didn't capture the malicious javascript but we know it was hosted on URL A and the end result was a redirect to URL B").

This seems like more of an issue with the docstring than the form. Maybe "A URL that redirects to another URL, such as via a URL shortening service or an HTTP 302 response."